### PR TITLE
ABB IRB1200 and Automate Cell

### PR DESCRIPTION
**Pull request by Jmeyer1292**
__Do not merge__: We still need to do a lot of testing with real hardware. I also need to sync changes to the Godel launch files.

This is for documentation's sake.

### CODE REVIEW

File: godel_robots/abb/godel_irb1200/abb_irb1200_support/package.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  <run_depend>abb_driver</run_depend>
+  <run_depend>joint_state_publisher</run_depend>
+  <run_depend>robot_state_publisher</run_depend>
+  <run_depend>rviz</run_depend>
+  <run_depend>xacro</run_depend>
+
+  <export>
+    <architecture_independent/>
+  </export>
+</package>
```
Comment (gavanderhoorn) [inline godel_robots/abb/godel_irb1200/abb_irb1200_support/package.xml:48]: Would it not be an idea to submit this to 'upstream' (ie: `ros-industrial/abb_experimental`)?

Comment (Jmeyer1292) [reply godel_robots/abb/godel_irb1200/abb_irb1200_support/package.xml:48]: @geoffreychiou Did the work, so I'd like him to submit it. Plus:
1. We haven't tested the real hardware yet
2. Already use a custom version of ABB (for kinetic)

Comment (gavanderhoorn) [reply godel_robots/abb/godel_irb1200/abb_irb1200_support/package.xml:48]: > We haven't tested the real hardware yet

not really a problem: as long as there is a reasonable trust in the faithfull modeling of the robot we can put it in `abb_experimental`.

> Already use a custom version of ABB (for kinetic)

this I don't understand. Support packages are about as distro-independent as they get, and I would say it's *always* better to merge things upstream than to have them in some fork.

Comment (Jmeyer1292) [reply godel_robots/abb/godel_irb1200/abb_irb1200_support/package.xml:48]: There are MoveIt plugins in the ABB repo that prevent the `indigo-devel` branch from compiling on Kinetic (the c++11 stuff).

I appreciate your concerns, but _I_ am not going to push it back right now. Others are welcome to do as they please.

Comment (gavanderhoorn) [reply godel_robots/abb/godel_irb1200/abb_irb1200_support/package.xml:48]: > There are MoveIt plugins in the ABB repo that prevent the `indigo-devel` branch from compiling on Kinetic (the c++11 stuff).

I know that. But that doesn't apply to a support pkg.

> I appreciate your concerns, but I am not going to push it back right now. Others are welcome to do as they please.

It was just a suggestion. I don't care either way.

**Comment by AustinDeric** [comment]
I recommend splitting the godel package (separating out godel_robots) with this robot which will also address #91.

**Comment by AustinDeric** [comment]
I have a PR against @Jmeyer1292's repo to add a visual model of the workcell

**Comment by AustinDeric** [comment]
Due to some errors i made in this branch @Jmeyer1292 and I determined it would be easier to do a new PR -> #202

**Pull request closed without merging**